### ISSUE TITLE
Scale may not be working correctly

### ISSUE THREAD

**Issue by Lughbaloo** [shudnal/HipLantern#1]
Really cool mod! I'm currently using version 1.0.2 on the Valheim 0.218.15 Ashlands release.

The Scale option in the config for local scale, set to default to 0.25, doesn't appear to be affecting the lantern while it is equipped. When the lantern is dropped on the ground, it does increase/decrease in size though. 
I do want to confirm that the scale option is intended for while it is equipped, is that right?
![image](https://github.com/shudnal/HipLantern/assets/44557173/19cb3e55-825f-47ca-90d0-5f15fd639dfe)

**Comment by shudnal** [comment]
You have to restart the game after you change that part of the config. As stated in config's description.
![image](https://github.com/shudnal/HipLantern/assets/20457922/a32d514c-6fd4-4563-a030-4fa7bc0ce49e)

You can adjust the values using RuntimeUnityEditor to get effect you need then apply the values to config and restart the game.

**Comment by Lughbaloo** [comment]
Ok I thought I was going crazy so I went back to test it and it kept happening. Then I realized I was having a similar issue with the quickstep mod and figured out it's because I'm making changes to the config while the game is still open and then restarting. Before I close the game, at some point the mod is overwriting my changes in the config. Reloading the config shows that the value was reset to 0.25. I double checked this theory this by closing the game completely, changing the value to 0.5 (It worked, hurray), launching the game, then changing the value to 1 and saving the file before restarting the game. The value was then overwritten in the config back to 0.5. 

So it's not a matter of restarting the game, it's a matter of making sure the game is closed while editing the config. Thanks!

Feel free to close this if you dont have any further comments.
